sci-mathematics/netgen/netgen-5.3.1.ebuild

```diff
@@ -0,0 +1,103 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
```

Comment:
Because you should always try to use the latest EAPI. Plus EAPI=5 is deprecated in the science overlay and only be used if there are incompatibilities with eclasses used.